integrations/third_party/git_sync/core/GitSyncManager.py

```diff
@@ -99,10 +99,23 @@ def __init__(
         self.content = GitContentManager(self.git_client, self.api)
 
     def __del__(self):
-        self.logger.info("Cleaning up")
-        if self.git_client:
-            self.git_client.cleanup()
-        self._wd.cleanup()
+        try:
+            if getattr(self, "logger", None):
```

Comment:
Should this also be like this?
```suggestion
            if getattr(self, "logger", None) is not None:
```